packages/cli/src/commands/setup/chakra-ui/tasks/setup-chakra.js

```diff
@@ -0,0 +1,95 @@
+import fs from 'fs'
+
+import { getPaths } from '../../../../lib'
+
+/**
+ * Transform an object to JSX props syntax
+ *
+ * @param obj {Record<string, any>}
+ * @param options {{exclude?: string[], raw?: boolean | string[]}}
+ * @returns {string[]}
+ */
+function objectToComponentProps(obj, options = { exclude: [], raw: false }) {
```

Comment:
@jtoar do we have any helpers/utils for adding a provider to App.js? Seems like we do this a few other places as well.

Fine either way — just attempting to simplify if possible.